src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -176,8 +176,10 @@ public void runAdditionalValidations(List<CastVoteRecord> castVoteRecords)
   }
 
   @Override
-  public boolean isRankingAllowed(int rank, String contestId) {
-    return rank > 0 && rank <= contests.get(contestId).getMaxRanks();
+  protected boolean isRankingAllowed(int rank, String contestId) {
+    return rank > 0
+      && rank <= contests.get(contestId).getMaxRanks()
+      && config.isRankingAllowed(rank);
```

Comment:
I'm having trouble understanding/remembering why the last condition isn't a sufficient check on its own here. At a minimum, I think the `rank > 0` part is redundant?

Reply:
Yes, `rank > 0` is redundant, though perhaps not obviously so? This checks: "Is this valid for this contest" (0 < rank < maxRanks), and separately, "is this valid in the config".

I don't mind removing `rank > 0`, but I can see an argument for leaving it in: it allows the config to define its own validity rules independent of what's valid for this dominion contest.